### ISSUE TITLE
(lack of) mobile support?

### ISSUE THREAD

**Issue by user670** [AlexNisnevich/untrusted#486]
The game seems to lack mobile support?

On the one hand I'd like to see the game get mobilee support; on the other hand if mobile support is not coming, detection of mobile device and a "this game can't be played on mobile" message would also be nice.

**Comment by darkwater4213** [comment]
Obviously, phones would not be a particularly viable option for playing untrusted—there’s just not enough real estate, so to speak. As for tablets (such as an iPad) it seems to work just fine. The vi-keys (hjkl, but not hjklyubn) work for me just fine for movement, and editing is cumbersome but not impossible (especially now that we can pull down on a key to access the next layer; one of apple’s best qol updates to date imo). As for a message… Fair point. It’s not immediately obvious that the page is going to behave as strangely as it does (thanks, apple… lol) but tablets should work fine.
